src/Bicep.Decompiler/DecompileOptions.cs

```diff
@@ -8,3 +8,8 @@ public record DecompileOptions(
     bool AllowMissingParamsAndVarsInNestedTemplates = false,
     bool IgnoreTrailingInput = true
 );
+
+public record DecompileParamOptions(
```

Comment:
should I move it into separate file?

Reply:
Probably, but whatever :-).  Less of an issue nowadays with Intellisense.